java-lotto/src/test/java/LottoTicketTest.java

```diff
@@ -15,13 +18,30 @@ class LottoTicketTest {
 
     @Test
     void 로또_일치_개수_계산_확인() {
-        LottoTicket ticket = new LottoTicket(List.of(1, 2, 3, 4, 5, 6));
-        int matchCount = ticket.getMatchCount(List.of(1, 2, 3, 10, 11, 12));
+        // 당첨 번호
+        List<LottoNo> winningNumbers = List.of(1, 2, 3, 4, 5, 6).stream()
+                .map(LottoNo::new)
+                .collect(Collectors.toList());
+        LottoNo bonus = new LottoNo(7);
+        WinningLotto winningLotto = new WinningLotto(new LottoTicket(winningNumbers, true), bonus);
+
+        // 테스트 티켓
+        List<LottoNo> testNumbers = List.of(1, 2, 3, 10, 11, 12).stream()
+                .map(LottoNo::new)
+                .collect(Collectors.toList());
+
+        LottoTicket ticket = new LottoTicket(testNumbers, true);
+        int matchCount = ticket.countMatchingNumbersWith(winningLotto.getWinningTicket());
```

Comment:
게터를 지양하고 메시지를 보내보면 어떨까요?

```suggestion
        int matchCount = ticket.countMatchingNumbersWith(winningLotto);
```

혹은 WinningLotto 에 책임을 더 부여해서 다음과 같이 사용할 수도 있구요
```java
Rank rank = winningLotto.match(ticket);
```